models/fixtures/training/bar.json

```diff
@@ -0,0 +1,93 @@
+{
+  "training": [
+    {
+      "role": "Bar",
+      "pass_auto": true,
+      "pass_mark": 100,
+      "url": "",
```

Comment:
Is there a missing value here?

Reply:
Yes, I'm not sure we know what's happening with the actual training material, that has previously been hosted elsewhere.

Reply:
When originally discussing this the URL was a config setting and I was told it was going to be rewritten:
```
16:13 < idn> Bar training is exploding at the moment, I have no idea what to do with that, but that's just me randomly clicking around. Should I card that?
16:13 < idn> KeyError: 'BAR_TRAINING_FORM'
16:14 < idn> Ah, missing config perhaps.
16:14 < russss> might be config again yeah...
16:14 < idn> Yeah, looks like it - url = app.config["BAR_TRAINING_FORM"]...
16:15 <@ms7821> bar training will be rewritten
16:15 < idn> ms7821: k, I'll ignore that then.
16:15 < idn> Ta :)
```
So at this point we don't have a URL to add.

Reply:
I would like the training material to be hosted on this website for next time. See "Training outline" in the Bar folder of the 2018 google drive. It may change before next time, it depends how the licence application process goes.

Reply:
Yup, but right now I'm focused on the test part.

Reply:
@sde1000 I don't think I have access to the Google Drive folder of which you speak.

Reply:
I'm assuming it's the same as this one though - https://usercontent.irccloud-cdn.com/file/3LYiO0Er/Training%20outline.pdf

Reply:
It is.

Reply:
If we could place a link to the original bar training document into this `url` JSON field (even if we change/remove it later), that'd keep me happy that we have a git-based record/reference to the source material.